libs/package.json

```diff
@@ -25,7 +25,7 @@
     "async-retry": "^1.2.3",
     "axios": "^0.19.0",
     "bs58": "^4.0.1",
-    "eth-sig-util": "^2.1.0",
+    "eth-sig-util": "^2.5.4",
```

Comment:
any reason not to pin this?

Reply:
i want to have a sep convo about pinning with the whole team. regardless of whether we pin, we should still try and stay on top of latest versions for security/compatibility reasons. I see them as mutually exclusive problems, basically. lmk what you think

Reply:
i feel like pinning is making a deal w/ the devil (albeit a probably good deal)